src/main/java/it/pn/frontend/e2e/pages/mittente/LoginPAPage.java

```diff
@@ -51,7 +51,7 @@ public void inserisciPassword(String pwd){
     }
 
     public void selezionaInviaDati(){
-        logger.info("Si selezione il buttone invia");
+        logger.info("Si selezione il bottone invia");
```

Comment:
correggere in si seleziona

Reply:
già fatto commit a649b30f7486424f149fb615d2da66844b6aa0fe